tests/components/homematicip_cloud/test_init.py

```diff
@@ -141,14 +97,17 @@
         instance.home.modelType = "mock-type"
         instance.home.name = "mock-name"
         instance.home.currentAPVersion = "mock-ap-version"
+        instance.async_reset.return_value = mock_coro(True)
 
         assert await async_setup_component(hass, hmipc.DOMAIN, {}) is True
 
-    assert len(mock_hap.return_value.mock_calls) >= 1
+    assert mock_hap.return_value.mock_calls[0][0] == "async_setup"
 
-    mock_hap.return_value.async_reset.return_value = mock_coro(True)
-    assert await hmipc.async_unload_entry(hass, entry)
+    config_entries = hass.config_entries.async_entries(hmipc.DOMAIN)
+    assert len(config_entries) == 1
+    assert await hmipc.async_unload_entry(hass, config_entries[0])
```

Comment:
Use the core interface here instead, `hass.config_entries.async_unload`.

https://github.com/home-assistant/home-assistant/blob/cd72128a8011369e1bf702d2b6af8f1f17543754/homeassistant/config_entries.py#L693

Reply:
Done